frontend/app/features/main/controllers/mainController.js

```diff
@@ -19,19 +20,56 @@
         }
     }
 
-    function shuffleArray(array) {
-        var currentIndex = array.length, temporaryValue, randomIndex;
-        // While there remain elements to shuffle...
-        while (0 !== currentIndex) {
-            // Pick a remaining element...
-            randomIndex = Math.floor(Math.random() * currentIndex);
-            currentIndex -= 1;
-            // And swap it with the current element.
-            temporaryValue = array[currentIndex];
-            array[currentIndex] = array[randomIndex];
-            array[randomIndex] = temporaryValue;
+
+// sort to show three past events
+    function findThreePastEvents (eventArray) {
```

Comment:
Please cooperate with @myshenin . You need to create 2 separate API  requests to get past and upcoming events